components/01-atoms/videos/_video.scss

```diff
@@ -0,0 +1,18 @@
+/* Responsive Video using CSS only */
+.video {
+  iframe {
+    height: 100%;
+    width: 100%;
+  }
+
+  video {
+    width: 100%;
+    height: auto;
+  }
```

Comment:
@callinmullaney I'm not quite sure where we fell on ordering styles, but maybe these (width, height) should be in the same order? 😃

Reply:
done